CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Code of Conduct
+permalink: /code-of-conduct/
+---
```

Comment:
Any particular reason for using static-site-generator metadata here? (it's Hugo meta, I believe?)

Reply:
Copy and paste :-), and left it in because I thought it might be important for GitHub ( ut obviously it's not).  Will snip this metadata.